arch/cortex-m4/src/lib.rs

```diff
@@ -193,11 +193,11 @@ pub unsafe extern "C" fn hard_fault_handler() {
     let kernel_stack: bool;
 
     asm!(
-        "mov    r1, 0                       \n\
-         tst    lr, #4                      \n\
-         itte   eq                          \n\
-         mrseq  r0, msp                     \n\
-         addeq  r1, 1                       \n\
+        "mov    r1, 0                       \r\n\
+         tst    lr, #4                      \r\n\
+         itte   eq                          \r\n\
+         mrseq  r0, msp                     \r\n\
+         addeq  r1, 1                       \r\n\
```

Comment:
Does this change do anything?

Reply:
lol no sorry. I got a little gung-ho adding carriage returns in this file and sprinkled it into some assembly it seems